kobo/hub/models.py

```diff
@@ -391,11 +391,6 @@ def _open_log(self, name):
         elif os.path.isfile(log_path + ".gz"):
             out = gzip.open(log_path + ".gz", "rb")
 
-            # GZipFile was not usable with BufferedReader
-            # until 2.7
-            if sys.version_info[0:2] >= (2, 7):
-                out = io.BufferedReader(out, LOG_BUFFER_SIZE)
```

Comment:
This change is removing python3 code, not python2 code.  i.e. the code on python3 has been using io.BufferedReader and this change would make it no longer do so.

Reply:
ah, I see. I read it wrong